### PR TITLE
Added watzon/tourmaline

### PR DESCRIPTION
**Pull request by watzon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -525,6 +525,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [raven.cr](https://github.com/sija/raven.cr) - Raven is a client for [Sentry](https://github.com/getsentry/sentry)
  * [slack.cr](https://github.com/DougEverly/slack.cr) - A Slack [Real Time Messaging API](https://api.slack.com/rtm) WebSocket client library
  * [telegram_bot](https://github.com/hangyas/telegram_bot) - A wrapper for the Telegram Bot API
+ * [tourmaline](https://github.com/watzon/tourmaline) - Telegram bot framework with an API loosely based on [telegraf.js](https://telegraf.js.org/)
```
Comment (veelenga) [inline README.md]: Let's move that to [Framework Components](https://github.com/veelenga/awesome-crystal#framework-components)

Comment (watzon) [reply README.md]: Done :)

**Comment by watzon** [comment]
Just realized that @hugoabonizio is from Londrina, BR. I lived there for 6 months lol.

**Comment by hugoabonizio** [comment]
@watzon that's so cool! Are you brazilian?

**Comment by watzon** [comment]
@hugoabonizio I'm not, but I lived in Brazil for a year. Foz do Iguacu first and then Londrina. I miss it and actually may be moving back down to South America soon if I get the job with Manas.

**Comment by hugoabonizio** [comment]
@watzon what a small world! I hope you get the job :smiley:

**Pull request merged**